crates/common/metrics/lean/compose.yml

```diff
@@ -0,0 +1,27 @@
+services:
+  prometheus:
+    image: prom/prometheus
+    container_name: prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yaml'
+    restart: unless-stopped
+    volumes:
+      - ./tools/prometheus:/etc/prometheus
+      - prometheus_data:/prometheus
+    ports:
+      - 9090:9090
+  grafana:
+    image: grafana/grafana
+    container_name: grafana
+    restart: unless-stopped
+    volumes:
+      - ./tools/grafana/datasource.yaml:/etc/grafana/provisioning/datasources/datasource.yaml
+      - ./tools/grafana/dashboard.yaml:/etc/grafana/provisioning/dashboards/main.yaml
+      - ./tools/grafana/dashboards:/var/lib/grafana/dashboards
+    ports:
+      - 3000:3000
+
+volumes:
+  prometheus_data:
+
+networks: {}
```

Comment:
please move the code being copy/pasted to `ream/metrics` it shouldn't be in this crate. 

<img width="450" height="402" alt="image" src="https://github.com/user-attachments/assets/bcde6064-e48a-499f-b87f-76a208561bcf" />

^ also I would match this folder structure instead of making a tools folder, as that folder would be redundent, once this isn't no longer in the crate.